__tests__/app.test.js

```diff
@@ -103,3 +103,29 @@ describe("Get /api/articles/:article_id", () => {
     })
 
 })
+
+describe("Get /api/articles", () => {
+    test("200: responds with an articles array of article objects with the relevant properties, sorted by date in descending order and without a body property", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toBeInstanceOf(Array)
```

Comment:
Nice idea with the below, but we could have used a forEach here to check that each element looks as expected to have better coverage.